src/presentation/settings/explorer.tsx

```diff
@@ -14,13 +13,17 @@ import {
   NigiriDefaultExplorerURLs,
 } from '../../domain/app';
 import SettingsCustomExplorerForm from '../components/explorer-custom-form';
+import { NetworkString } from 'ldk';
+import { appInitState } from '../../application/redux/reducers/app-reducer';
 
-function explorerTypesForNetwork(network: Network): ExplorerType[] {
+function explorerTypesForNetwork(network: NetworkString): ExplorerType[] {
   switch (network) {
     case 'liquid':
       return ['Blockstream', 'Mempool', 'Custom'];
     case 'regtest':
       return ['Nigiri', 'Custom'];
+    case 'testnet':
```

Comment:
Would move this after Liquid, so in the UI in Settings we have them sorted

<img width="314" alt="image" src="https://user-images.githubusercontent.com/3596602/144837602-01c32b41-3e35-4f19-87a3-2262bce02a00.png">